### PR TITLE
Refactor/error handling accessibility and minor fixes

### PR DESCRIPTION
**Pull request by kpn678**
Hello, team! Please review this PR!

Where should your reviewer start?
- Line number 57 on index.html

What (if any) features are you implementing?
- Ensure that data type is numbers when inputting POST information
- Make images tab-able

What (if anything) did you refactor?
- Combined lines of code for first .then in post data functions into a separate throw error function

Were there any issues that arose?
- Tried changing the name of the class of the submit button on the POST modal so that required attributes will work, but still not working

Thank you!!

### CODE REVIEW

**Comment by kpn678** [comment]
Adding onto the last PR:

- The date is now reflecting the correct date because of taking out the new Date instance
- Required in the input fields is also working as intended because of putting the event on the overall form instead of the submit button
- While page is tab-able, we cannot hit enter on the buttons

File: src/scripts.js [hunk 1]
```diff
@@ -93,7 +94,7 @@ window.onclick = function(event) {
   };
 };
 
-postSubmit.addEventListener("click", (event) => {
+addDataForm.addEventListener("submit", (event) => {
```
Comment (jordan-sullivan) [inline src/scripts.js:97]: interesting. -J

**Review by jordan-sullivan — APPROVED** [review]
Interesting fixes, Glad we reached out to Travis. Thanks for getting this finished.

**Comment by GrantXBeard** [comment]
agreed. glad we could get some clarity here.

**Pull request merged**